Admin/AdminHelper.php

```diff
@@ -284,13 +285,11 @@ public function getElementAccessPath($elementId, $entity)
                     $partReturnValue .= $dot.$id;
                     $dot = '.';
                     $id = '';
-                } else {
-                    $dot = '';
                 }
             }
 
-            if ($dot !== '.') {
-                throw new \Exception(sprintf('Could not get element id from %s Failing part: %s', $elementId, $subValue));
+            if (!empty($id)) {
```

Comment:
More or less...

The strings passed to this methods looks like: SomeId_attribute1_attribute2_attribute3part1_attribute3part2 ...etc

This issue is all about using the same separator '_' can be used both for the property separator and in a property name. This method can't guess automagically how the property is called so it will try to match all the string. Basically is trying to translate things like this: attribute1_attribute2_attribute3part1_attribute3part2 in attribute1.attribute2.attribute3part1_attribute3part2

So yes, the last iteration must go well, because if not the id will be 'not empty' and then will throw an exception.